### PR TITLE
Normalize whitespace

### PR DESCRIPTION
**Pull request by NathanBaulch**
Removes trailing whitespace and superfluous new lines.

### CODE REVIEW

File: docs/mem.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
-# mem
+#vfsSimpleSuitemem
```
Comment (funkyshu) [inline docs/mem.md:1]: This seems like a mistake.

**Review by funkyshu — CHANGES_REQUESTED** [review]
Add an entry to CHANGELOG.md.

File: docs/mem.md (hunk 1 above)
Comment (NathanBaulch) [reply docs/mem.md]: Oops, fixed.

File: backend/testsuite/io_integration_test.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ func (o *OSWrapper) Read(b []byte) (int, error) {
 		return 0, errors.New("file not found")
 	}
 	if o.file == nil {
-		file, err := os.OpenFile(o.filename, os.O_RDWR, 0600)
+		file, err := os.OpenFile(o.filename, os.O_RDWR, 0o600)
```
[the remark this answers is not in the document]
Comment (NathanBaulch) [reply backend/testsuite/io_integration_test.go]: Not a typo, this was a side-effect of using my preferred (and highly opinionated) code formatting tool [gofumpt](https://github.com/mvdan/gofumpt).

**Review by funkyshu — APPROVED** [review]

**Pull request merged**